charts/icm-as/templates/_helpers.tpl

```diff
@@ -61,3 +61,17 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the values for the environment variable ADDITIONAL_VM_PARAMETERS
+*/}}
+{{- define "icm-as.additionalVMParameters" -}}
```

Comment:
For me this is too laborious. If somebody wants to enable datadog he should just set the following in the values.yaml:

```
jvm:
  additionalOptions: -Ddd.profiling.enabled=true

datadog:
  enabled: true
```

Other wise this would get too complex. There is no need for an entry `_helpers.tpl`.
Or do you have some other options where this would makes sense?